packages/backend-storage/src/index.ts

```diff
@@ -1,9 +1,3 @@
 export * from './factory.js';
 export { StorageResources, AmplifyStorageProps } from './construct.js';
-export {
-  RoleAccessBuilder,
-  StorageAccessDefinition,
-  StorageAccessBuilder,
-  StorageAction,
-} from './access_builder.js';
```

Comment:
Doesn't this result in a change to `API.md`? Might need to update.

Reply:
I just moved these to a different file. There's no change to the API

Reply:
Ah, missed those at the bottom of types